virttest/utils_conn.py

```diff
@@ -1299,6 +1299,10 @@ def server_setup(self, on_local=False):
                         cakey_path: self.pki_CA_dir,
                         servercert_path: self.libvirt_pki_dir,
                         serverkey_path: self.libvirt_pki_private_dir}
+            cmd = "ls {0} > /dev/null 2>&1 || mkdir -p {0}".format(self.pki_CA_dir)
```

Comment:
Do we need 'ls {0} > /dev/null 2>&1 ||'? In my test, 'mkdir -p' will not report an error if the directory exists already.

Reply:
yes, it works even if we don't check the dir's existence.
I added this check because making  pki_CA_dir look like a mandatory step. Do you want me to change it?

Reply:
I would suggest to remove the ls part as it makes the step more complicated/less readable with no real effect especially when you throw all the ls cmd output away.

Reply:
@kamilvarga @dzhengfy Thank you for your comment! I agree it was a complicated command. Removed. 
Would you please re-review?